3g-watchdog_test.py

```diff
@@ -144,9 +145,11 @@ def old_hardware(mocker):
 @pytest.fixture(autouse=True)
 def modem(mocker):
     m = mocker.patch.object(watchdog, "Modem")
-    m.NETDEV = "usb0"
-    m.is_present.return_value = True
-
+    m.return_value = mock.Mock()
+    instance = m.return_value
+    instance.netdev = "usb0"
+    instance.is_present.return_value = True
+    return instance
```

Comment:
It occurs to me that we get test more of the code by just patching a few methods on `Modem` instead patching the whole class. No need to do that right now but something to consider later.